radix-engine-constants/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 //==========================
 
 /// The default cost unit limit.
-pub const DEFAULT_COST_UNIT_LIMIT: u32 = 100_000_000;
+pub const DEFAULT_COST_UNIT_LIMIT: u32 = 500_000_000;
```

Comment:
Downstream has been relying on it being 100M for now. Is there a particular reason that we want to increase it even more, before metering fine-tuning?

Reply:
Yes, due to the increased number of substate locks/reads our `large_package` test is failing. I can fix this by reducing the cost units of those.